postgres/content.md

```diff
@@ -40,6 +40,29 @@ postgres=# SELECT 1;
 
 Run `docker stack deploy -c stack.yml %%REPO%%` (or `docker-compose -f stack.yml up`), wait for it to initialize completely, and visit `http://swarm-ip:8080`, `http://localhost:8080`, or `http://host-ip:8080` (as appropriate).
 
+## Healtcheck
+
+To add a health check to the image use the following command:
+
+```console
+docker run --name some-postgres -e POSTGRES_PASSWORD=mysecretpassword \
+	--health-cmd='pg_isready' --health-interval=30s -d %%IMAGE%%
+```
+
+or using docker-compose
+
+```yaml
+services:
+  postgres:
+    image: postgres
+    restart: always
+    environment:
+      POSTGRES_PASSWORD: example
+    healthcheck:
+        test: pg_isready
+        interval: 30s
```

Comment:
What do you think of making this `interval` smaller?

Reply:
Certainly. I don't know what would be a good balance.

Reply:
I am rolling with 2 seconds, and think in general an example may be better served by 5-sec or 10-sec.  I defer to your judgement here beyond this, and thanks for doing this!

Reply:
5s seems reasonable.